packages/clerk-js/src/ui/localization/defaultEnglishResource.ts

```diff
@@ -261,13 +280,35 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
         resendButton: 'Resend link',
         successMessage: 'The email {{identifier}} has been added to your account.',
       },
+      removeResource: {
+        title: 'Remove email address',
+        successMessage: '{{emailAddress}} has been removed from your account.',
+      },
+    },
+    phoneNumberPage: {
+      title: 'Add phone number',
+      successMessage: '{{identifier}} has been added to your account.',
+      removeResource: {
+        title: 'Remove phone number',
+        successMessage: '{{phoneNumber}} has been removed from your account.',
+      },
     },
     connectedAccountPage: {
       title: 'Add connected account',
       formHint: 'Select a provider to connect your account.',
       formHint__noAccounts: 'There are no available external account providers.',
       socialButtonsBlockButton: 'Connect {{provider|titleize}} account',
       successMessage: 'The provider has been added to your account',
+      removeResource: {
+        title: 'Remove connected account',
+        successMessage: '{{connectedAccount}} has been removed from your account.',
+      },
+    },
+    web3WalletPage: {
```

Comment:
Good thinking reusing the key from the section (+) button, but let's add a title explicitly for 2 reasons:
- All pages should share the same API as much as possible. Making the API consistent for the end user is very important.
- We'd like to have separate keys, otherwise the users won't be able to have different strings for the button and the page title